src/content/blog/2024/special_interest_groups.mdx

```diff
@@ -0,0 +1,155 @@
+
+# Background
+
+Until now, nf-core has been very focused on doing what we do best:
+creating analysis pipelines and the tooling around them.
+We have structured ourselves around this: Slack channels for every pipeline
+bring together people working on the same set of tools to collaborate.
+It's clear to newcomers where to find help and it keeps the discussion focussed.
+
+However, as the community grows, the tool-specific organisation is beginning to show a few gaps.
+Different research projects can use the _same_ set of pipelines in very different ways:
+for example [nf-core/sarek](/sarek) was primarily written for analysis of somatic
+cancer sequencing analysis, with tumor:normal Human DNA samples as inputs.
+However, it works great as a generic variant calling pipeline and indeed many groups use
+it with DNA sequencing data from a diverse range of organisms, even bacteria!
+Most nf-core pipelines are very flexible in their usage by design, and so for truly
+reproducible research groups need to standardise not just the pipelines that they
+use, but also the manner in which they use them.
+
+We see more and more large research consortia adopting nf-core pipelines for their
+research and these concerns are often key to their desire to make research outputs
+from disparate groups comparable and reproducible.
+
+To meet this need, we want to provide a space for community members working on
+similar research interests to gather and collaborate.
+We want to foster collaboration not just in the development of scientific
+research tooling, but also in the usage of that tooling across multiple fields.
+We hope that special interest groups will provide a place for this to happen.
+
+# How will this work?
+
+We like how our community is structured around pipelines.
+Our established methods for these groups appears to work well, so rather than
+reinventing the wheel we intend to adopt a similar pattern for special interest groups:
+
+1. Anyone can suggest a new group in the `#new-interest-groups` Slack channel
+    - Just like with `#new-pipelines`, there is a Slack workflow with a form to fill in,
+      with a handful of questions to collect all the required information.
+2. The core team (and steering group, as necessary) will discuss the proposal, suggest
+   any changes and either accept or reject the proposal.
+    - This process will be tracked in a GitHub project, called
+      [New interest groups](https://github.com/orgs/nf-core/projects/63)
+3. A new Slack channel will be created for the new interest group on the nf-core Slack
+4. A new web page will be created for the group on the nf-core website
+5. The lead for the new group will organise a nf-core/bytesize talk to introduce
+   the group to the community. A blog post will also be recommended.
+6. The new special interest group will be active and continue under their own steam.
+
+## Group web pages
+
+The new web page will just be a stub at first, but we hope that the interest group
```

Comment:
Is this ready? If yes would be nice to include a pointer to where this will live.